From 683f98900e710712c397c8fb6b3571b776875088 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 18 Nov 2024 11:59:39 +0100
Subject: [PATCH 1372/1695] dts: stm32: do not expose CA35 regulators to CM33
 for stm32mp21x

Theses regulators should not be exposed to CM33 as same as STM32MP25x and
STM32MP23x families.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git d8243ca07f1521663cc22319de2841db63765e08 dts: stm32: do not expose CA35 regulators to CM33 for stm32mp21x]
Fixes: ce64cb4654 ("dts: stm32: describe scmi resources for stm32mp21x")
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I40691c10456da2b48d89fc1cb4be285802ff94b0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/419305
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 20 --------------------
 1 file changed, 20 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 8e42d370b..36a96aaec 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -1941,26 +1941,6 @@
 				regulators {
 					#address-cells = <1>;
 					#size-cells = <0>;
-
-					voltd-vddio1 {
-						reg = <VOLTD_SCMI_VDDIO1>;
-						voltd-supply = <&vddio1>;
-					};
-
-					voltd-vddio2 {
-						reg = <VOLTD_SCMI_VDDIO2>;
-						voltd-supply = <&vddio2>;
-					};
-
-					voltd-vddio3 {
-						reg = <VOLTD_SCMI_VDDIO3>;
-						voltd-supply = <&vddio3>;
-					};
-
-					voltd-vdda18adc {
-						reg = <VOLTD_SCMI_ADC>;
-						voltd-supply = <&vdda18adc>;
-					};
 				};
 			};
 		};
-- 
2.47.2

